WalletWasabi/Blockchain/TransactionBuilding/TransactionModifierWalletExtensions.cs

```diff
@@ -280,6 +282,24 @@ public static BuildTransactionResult CpfpTransaction(this Wallet wallet, SmartTr
 		var destination = keyManager.GetNextChangeKey().GetAssumedScriptPubKey().GetDestinationAddress(network);
 		Guard.NotNull(nameof(destination), destination);
 
+		// Request the unconfirmed transaction chain so we can extract the fee paid by tx + all the ancestors still unconfirmed.
+		using var cts = new CancellationTokenSource(TimeSpan.FromSeconds(20));
+		EffectiveTransactionStatus? unconfirmedTransactionChain = null;
```

Comment:
We have to review the naming because it is really hard to review for me, it is like nothing makes sense.